Backpack/src/main/java/net/skyscanner/backpack/fab/BpkFab.kt

```diff
@@ -4,28 +4,22 @@ import android.animation.AnimatorInflater
 import android.content.Context
 import android.content.res.ColorStateList
 import android.graphics.Canvas
-import android.graphics.Color
-import android.graphics.Paint
 import android.graphics.drawable.Drawable
-import android.graphics.drawable.RippleDrawable
-import android.graphics.drawable.ShapeDrawable
-import android.graphics.drawable.shapes.OvalShape
 import android.util.AttributeSet
 import androidx.annotation.ColorInt
-import androidx.appcompat.widget.AppCompatButton
 import androidx.core.content.ContextCompat
 import androidx.core.graphics.drawable.DrawableCompat
+import com.google.android.material.floatingactionbutton.FloatingActionButton
 import net.skyscanner.backpack.R
 import net.skyscanner.backpack.util.BpkTheme
 import net.skyscanner.backpack.util.createContextThemeWrapper
-import net.skyscanner.backpack.util.darken
 import net.skyscanner.backpack.util.use
 
 open class BpkFab @JvmOverloads constructor(
```

Comment:
Please delegate the attributes to FloatingActionButton implementation.

Reply:
👍